weave-js/src/components/PagePanelComponents/Home/Browse3/pages/LeaderboardPage/LeaderboardDeleteButton.tsx

```diff
@@ -0,0 +1,66 @@
+import {Button} from '@wandb/weave/components/Button';
+import React, {useContext, useState} from 'react';
+import {useHistory} from 'react-router-dom';
+
+import {
+  useClosePeek,
+  useWeaveflowRouteContext,
+  WeaveflowPeekContext,
+} from '../../context';
+import {DeleteModal} from '../common/DeleteModal';
+import {useWFHooks} from '../wfReactInterface/context';
+
+export const DeleteLeaderboardButton: React.FC<{
```

Comment:
can we please remove the delete button? This is such a destructive operation and could very easily be accidentally clicked and erase a bunch of config work. We can separate adding a delete button from the edit button in peak

Reply:
if it has a confirmation modal, then I am fine with it. In general, i think this action should be very challenging to do and hidden. But as long as it is not a single click, i am fine